entities/reader.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../utils/check'
+# frozen_string_literal: true
+
+# Class that describes reader object
+class Reader
+  attr_reader :name, :email, :city, :street, :house
+  include Check
+
+  def initialize(name, email, city, street, house)
+    empty?(name, email, city, street)
+    string?(name, email, city, street)
+    positive?(house)
+    integer?(house)
+
+    @name = name
```

Comment:
misslogic
if house will be a String your method ```positive?``` raise exception 
positive?(house)
integer?(house)
replace
integer?(house)
positive?(house)